price_feeds_poller.py

```diff
@@ -115,11 +115,121 @@ def handle_requestUpdate(
         print(f" ==== Previous query id : {feed_latest_update_query_id}\n")
         return [ feed_latest_update_query_id, tx.hex(), total_fee ]
 
+def reload_config(config_file_path, network_name):
+    config = load_dfe_config(config_file_path)
+    network_config = get_network_config(config, network_name)
+    if network_config['version'] != '2.0':
+      print(f"Fatal: Network '{network_name}' not prepared for 2.0")
+      exit(1)
+    return config, network_config.get('address', config['contracts']['2.0']['address'])
+
+def reload_pfs(feeds, config, network_name):
+    
+  captionMaxLength = 0
+  ids = []
+  pfs = []    
+
+  supports = feeds.functions.supportedFeeds().call()
+  for index in range(len(supports[0])):
+    caption = supports[1][index]
+    pf_id = supports[0][index].hex()
+    rad_hash = supports[2][index].hex()
+      
+    print(f"{caption}:")
+    routed = rad_hash.startswith("0000000000000000000000000000000000000000")
+    if routed == False:
+      cooldown = get_price_feed_config(config, network_name, caption, "minSecsBetweenUpdate", 3600)
+      deviation = get_price_feed_config(config, network_name, caption, "deviationPercentage", 3.5)
+      heartbeat = int(get_price_feed_config(config, network_name, caption, "maxSecsBetweenUpdates", 86400))
+      for attempt in range(5):
+        try:    
+          if routed == False:
+            bytecode = feeds.functions.lookupWitnetBytecode(pf_id).call()  
+          else:
+            bytecode = ""
+          latest_price = feeds.functions.latestPrice(pf_id).call()
+          latest_update_query_id = feeds.functions.latestUpdateQueryId(pf_id).call()
+          pending_update = latest_price[3] == 1
+          ids.append(pf_id)
+          pfs.append({
+            "id": pf_id,
+            "bytecode": bytecode,
+            "caption": caption,
+            "cooldown": cooldown,
+            "deviation": deviation,
+            "heartbeat": heartbeat,
+            "isRouted": routed,
+            "latestPrice": latest_price[0],
+            "latestTimestamp": latest_price[1],
+            "latestUpdateQueryId": latest_update_query_id,
+            "pendingUpdate": pending_update,
+            "radHash": rad_hash,
+            "revert": 0,
+            "auto_disabled": False,
+            "lastRevertedTx": "",
+            "lastUpdateFailed": False,
+            "lastUpdateFailedTimestamp": int(time.time()),
+            "fees": [],
+            "secs": []
+          })
+          print(f"  => ID4         : {pf_id}")
+          if routed == True:
```

Comment:
Are we modifying the `routed` value? if not, this `if` statement seems unnecessary